### ISSUE TITLE
🐞 [BUG] - Want to limit the digits that can be added to phone number in contact us form and provide link of Blogs to Blogs button in Home page 

### ISSUE THREAD

**Issue by Sar-thak-3** [amupedia2021/Project-Amupedia#315]
### Description

I want to add the limit of digits that can be entered in the Phone number column of the Contact Us form and also add the link of Blogs to blogs button in home page

### Current behaviour

-

### Expected behaviour

-

### Reproduction steps

```bash
1. Go to the Contact Us form
4. See the error
```

### Screenshots

![image](https://github.com/amupedia2021/Project-Amupedia/assets/100359818/86a48a9e-618e-43d9-b3b4-a3b5db400c80)

### Browsers

Firefox, Chrome, Safari, Microsoft Edge, Opera

### Are you contributing for a program ?

GSSoC2023

### Code of Conduct

- [X] I have read and agree to the Contributing Guidelines of this project.

**Comment by tusharbansal22** [comment]
Yep, validation is required in both phone number and email sections. If you are doing please use RegEx. Just want to confirm if it is assigned to someone else @AmanJain18 @anmode ?

Also will you be able to implement realtime validation?

**Comment by tanwarAalok** [comment]
@Sar-thak-3 Can you assign this issue to me under GSSOC 23 ?

**Comment by AmanJain18** [comment]
Issue already got fixed... check pr #335
